deployments/systemd/packages/Makefile

```diff
@@ -29,6 +29,10 @@ endif
 # strip 'v' from version string
 PACKAGE_VERSION := $(VERSION:v%=%)
 
+# replace '-' with '~' in the version string
+# e.g. 'x.y.z-rc.1' gets represented as 'x.y.z~rc.1'
+PACKAGE_VERSION := $(subst -,~,$(PACKAGE_VERSION))
```

Comment:
Why are we doing this btw?

Reply:
'-' is an illegal character for RPM package versioning

Reply:
Ahh got it